instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Comment from './Comment'
+import CommentBox from './CommentBox'
+import styled from 'styled-components'
+
+class CommentSection extends React.Component {
+    state = {
+        comments: this.props.comments,
+    }
+    
+    addNewComment = (newValue) => {
+        this.setState ({
+            comments: [...this.state.comments, {id: Date.now(), username: 'NuckChorris', text: newValue}]
```

Comment:
This would be a great place to make the username more dynamic by grabbing it for localStorage.